<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpringWidget</class>
 <widget class="QWidget" name="SpringWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpringWidget</string>
  </property>
  <widget class="QChartView" name="graphicsView_y">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="graphicsView_v">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="graphicsView_a">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="graphicsView_f">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>276</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_restart">
      <property name="text">
       <string>重新开始</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>324</width>
     <height>146</height>
    </rect>
   </property>
   <property name="title">
    <string>设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>小球质量</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_m"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>弹性系数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_kSpring"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>阻尼系数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_damping"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
